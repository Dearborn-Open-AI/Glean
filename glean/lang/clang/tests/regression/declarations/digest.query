query: digest.FileDigest _
transform: [gensort, []]
