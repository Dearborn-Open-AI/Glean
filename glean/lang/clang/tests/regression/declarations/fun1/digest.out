[
  "@generated",
  {
    "key": {
      "file": { "key": "test.cpp" },
      "digest": {
        "hash": "eb09205f432b77d2baba877b0dd2f9babc3a6246",
        "size": 310
      }
    }
  }
]